pkgs/development/python-modules/colorcet/default.nix

```
@@ -22,6 +23,10 @@ buildPythonPackage rec {
     pyct
   ];
 
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
```

```suggestion
  build-system = [
    setuptools
    setuptools-scm
  ];
```

Because setuptools-scm currently propagates setuptools, but that may change.

---

Done!